config/webpack/common.webpack.config.js

```diff
@@ -48,6 +48,56 @@ function getWebpackConfig(skyPagesConfig, argv = {}) {
       break;
   }
 
+  let plugins = [
+    // Some properties are required on the root object passed to HtmlWebpackPlugin
+    new HtmlWebpackPlugin({
+      template: skyPagesConfig.runtime.app.template,
+      inject: skyPagesConfig.runtime.app.inject,
+      runtime: skyPagesConfig.runtime,
+      skyux: skyPagesConfig.skyux
+    }),
+
+    new CommonsChunkPlugin({
+      name: ['skyux', 'vendor', 'polyfills']
+    }),
+
+    new webpack.DefinePlugin({
+      'skyPagesConfig': JSON.stringify(skyPagesConfig)
+    }),
+
+    new LoaderOptionsPlugin({
+      options: {
+        context: __dirname,
+        skyPagesConfig: skyPagesConfig
+      }
+    }),
+
+    new ContextReplacementPlugin(
+      // The (\\|\/) piece accounts for path separators in *nix and Windows
+      /angular(\\|\/)core(\\|\/)@angular/,
+      spaPath('src'),
+      {}
+    ),
+
+    // Webpack 2 behavior does not correctly return non-zero exit code.
+    new ProcessExitCode(),
+
+    new OutputKeepAlivePlugin({
+      enabled: argv['output-keep-alive']
+    })
+  ];
+
+  // Supporting a custom logging type of none
+  if (argv.log !== 'none') {
+    const log = skyPagesConfig.runtime.command === 'serve' ? 'compact' : 'expanded';
```

Comment:
Is the intent that `skyux build` should produce the "expanded" logs? I use `skyux build -s` often, and it add a ton of logs to the terminal. Curious if you thought about making the log level default to "compact" and put the option in skyuxconfig.json instead? This would allow control of the log level globally, but could be overwritten in command-specific config files.

Reply:
Apologies as that is a bit vague.  `compact` and `expanded` produce the same logging, only difference is compact does it all on a single line.  This is good for local development, but of any "CI" it would not be a good experience.  https://github.com/dominique-mueller/simple-progress-webpack-plugin#compact

I am making the assumption here that if you're running build, you're probably on a CI.  Obviously, you can run build locally, but seemed like the safer default to set.

Allowing the default to be set in skyuxconfig.json might not be a bad idea, as long as the command line overrides it.  This would mean users could get verbose logging on the build machine without altering their build definition, which would be nice.